### DISCUSSION TITLE
Thanks openf1! Just been having fun using openf1 API and built a Shiny app with Python!

### DISCUSSION THREAD

**Discussion by kleu046** [br-g/openf1#44] [Show and tell]
Learning Shiny with Python.  Thanks to openf1's great work, I was able to combine the my two passions - F1 and data visualisation into one!

I have also been a FastF1 user too!  Having used that makes openf1 really familiar and easy to use!

p.s. I should say I borrowed some visualisation ideas from F1 TV as a learning exercise too!  ;)

<img width="1641" alt="image" src="https://github.com/br-g/openf1/assets/89670261/99e5d014-9aec-4d0b-b0d2-71dbfa206e2e">

**Comment by br-g** [comment]
Awesome work! Thank you for sharing! 😊

**Comment by EthanKyleKim** [comment]
How did you create the map?

Did you use the x, y, z data from the location?
